app/components/traffic-measure/index.hbs

```diff
@@ -82,6 +82,48 @@
           <VariableSignageSelector @concept={{@trafficMeasureConcept}} />
         </div>
       </div>
+      {{#let @trafficMeasureConcept.error.startDate as |error|}}
+          {{#let @trafficMeasureConcept.warning.startDate as |warning|}}
+            <AuFormRow>
+              <AuLabel
+                @error={{error}}
+                @warning={{warning}}
+                for="startDate"
+              >
+                {{t "utility.start-date"}}&nbsp;
+              </AuLabel>
+              <AuDatePicker
+                @error={{error}}
+                @warning={{warning}}
+                id="startDate"
+                @value={{@trafficMeasureConcept.startDate}}
+                @onChange={{fn this.setTrafficMeasureDate 'startDate'}}
+              />
+              <ErrorMessage @error={{error}} @warning={{warning}} />
+            </AuFormRow>
+          {{/let}}
+        {{/let}}
+        {{#let @trafficMeasureConcept.error.endDate as |error|}}
+          {{#let @trafficMeasureConcept.warning.endDate as |warning|}}
```

Comment:
I think it should be possible to merge these two lets into one statement, which should reduce the nesting a bit.